src/generated/cloud/beyondcorp/clientconnectorservices/v1/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+[package]
+name                 = "google-cloud-beyondcorp-clientconnectorservices-v1"
+version              = "0.2.0"
+description          = "Google Cloud Client Libraries for Rust - BeyondCorp API"
```

Comment:
Will this be repeated for more libraries under `google/cloud/beyondcorp`?

Reply:
Unless they changed their title in the service config YAML, yes.  They will all have the same title.
We /could/ override the title, now or later.

Reply:
Moved that commit to a separate PR, where it belongs.